### ISSUE TITLE
RuboCop fixes multiline ternary operators resulting in incorrect indentation.

### ISSUE THREAD

**Issue by ioquatix** [rubocop-hq/rubocop#8575]
Given the following input:

``` ruby
module M
  def foo
    bar ?
      10 :
      20
  end
end
```

## Expected behavior

``` ruby
module M
  def foo
    if bar
      10
    else
      20
    end
  end
end
```

## Actual behavior

``` ruby
module M
  def foo
    if bar
      10
    else
      20
end
  end
end
```

I have seen other permutations of the wrong output e.g. the else statement might also be indented incorrectly.

## Steps to reproduce the problem

Put the sample into a file `test.rb` and run `rubocop -a` in the same directory.

## RuboCop version

```
koyoko% rubocop -V
0.89.1 (using Parser 2.7.1.4, rubocop-ast 0.3.0, running on ruby 2.7.1 x86_64-linux)
```

**Comment by ioquatix** [comment]
![image](https://user-images.githubusercontent.com/30030/91004403-31fd8500-e628-11ea-9a80-d1b57742b9e2.png)

I see it identified the end indentation issue. But it didn't fix it. Can I auto fix it? I will try now.

**Comment by koic** [comment]
Can you set `true` to `Lint/EndAlignment`cop's `AutoCorrect` option?

```yaml
# .rubocop.yml
Lint/EndAlignment:
  Enabled: true
  AutoCorrect: true
```

I remembered the same issue I had with the Rails repository before.
https://github.com/rails/rails/pull/31732

**Comment by ioquatix** [comment]
Thank you so much. That seems to fix it in test case. Let me check main code base (private).

**Comment by ioquatix** [comment]
I checked, and with full `rubocop -a` it fixed the issue.

**Comment by koic** [comment]
JFYI, the `AutoCorrect` config is now `true` by default. It will be available in the next RuboCop release (maybe 0.90). Thank you.
#8594
